src/UI/ListView/SummaryItem.elm

```diff
@@ -103,6 +112,39 @@ withSelected selected (SummaryItem prop opt) =
     SummaryItem prop { opt | selected = selected }
 
 
+{-| Adds an icon to the right side of the summary.
+
+    Summary.withIcon (Icon.fix "Problem Solving")
+        someListView
+
+-}
+withIcon : Icon -> SummaryItem msg -> SummaryItem msg
+withIcon icon (SummaryItem prop opt) =
+    SummaryItem prop { opt | icon = Just icon }
```

Comment:
if we see ourselves changing the API in the future to receive yet another thing to be put to the right, perhaps we should start calling it an "addon" and just receive any element